pxr/imaging/plugin/hdRpr/CMakeLists.txt

```diff
@@ -20,6 +20,7 @@ if(HoudiniUSD_FOUND)
     list(APPEND OptLibs Houdini)
     add_definitions(-DENABLE_MULTITHREADED_RENDER_BUFFER)
     add_definitions(-DHDRPR_DEFAULT_MATERIAL_NETWORK_SELECTOR="karma")
+    add_definitions(-DHOUDINI_BUILD)
```

Comment:
You can use `BUILD_AS_HOUDINI_PLUGIN` instead of introducing another definition - https://github.com/GPUOpen-LibrariesAndSDKs/RadeonProRenderUSD/blob/master/pxr/imaging/rprUsd/CMakeLists.txt#L52